client/src/test/formatting.test.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+import * as assert from 'assert';
+import { getDocUri, activate} from './helper';
+
+suite('Should get diagnostics', () => {
+	test('formatting', async () => {
+		await testformatting(getDocUri('formatting/before_formatting.ion'), getDocUri('formatting/after_formatting.ion'));
+	});
+});
+
+async function testformatting(docUri_BF: vscode.Uri, docUri_AF: vscode.Uri) {
+	await activate(docUri_BF);
+	const textDocument_BF: vscode.TextDocument = vscode.window.activeTextEditor.document; 
+	// console.log ('Raw file ->\n', textDocument_BF.getText ()); 
```

Comment:
Please remove debugging code from PRs.

Reply:
I thought commenting them out will also work.

Reply:
Typically for code that we plan to release we remove debugging statements whether they are in comments or not. They clutter and distract when reading the code.